src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -11,8 +11,8 @@ public class Phone {
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers should only contain numbers, and it should be at least 8 digits long";
+    public static final String VALIDATION_REGEX = "\\d{8,}";
```

Comment:
Are we allowed to modify this feature's behaviour at this stage? Accepting phone numbers that are <8 digits long might be considered a feature flaw. Feature flaws can be fixed by stating plans to fix them in the Planned Enhancements section of our DG.

Reply:
You're right! Nice catch and thanks for informing.